pkg/operator/context/apis.go

```diff
@@ -25,20 +25,16 @@ import (
 	"github.com/cortexlabs/cortex/pkg/lib/errors"
 	"github.com/cortexlabs/cortex/pkg/lib/hash"
 	"github.com/cortexlabs/cortex/pkg/lib/pointer"
-	"github.com/cortexlabs/cortex/pkg/lib/sets/strset"
 	s "github.com/cortexlabs/cortex/pkg/lib/strings"
 	"github.com/cortexlabs/cortex/pkg/operator/api/context"
 	"github.com/cortexlabs/cortex/pkg/operator/api/resource"
 	"github.com/cortexlabs/cortex/pkg/operator/api/userconfig"
 	"github.com/cortexlabs/cortex/pkg/operator/config"
 )
 
-var uploadedRequestHandlers = strset.New()
-
 func getAPIs(config *userconfig.Config,
 	deploymentVersion string,
 	impls map[string][]byte,
-	pythonPackages context.PythonPackages,
 ) (context.APIs, error) {
```

Comment:
I believe that we need to add project ID to the API ID, so that if we change a python file it gets re-deployed. Can you test this with the encoder file you split out?